prime-router/src/main/resources/metadata/hl7_mapping/receivers/STLTs/IL/IL-receiver-transform.yml

```diff
@@ -0,0 +1,70 @@
+hl7Class: ca.uhn.hl7v2.model.v251.message.ORU_R01
+
+extends: classpath:/metadata/hl7_mapping/ORU_R01/ORU_R01-base.yml
+
+elements:
+
+  - name: xtn-use-code-from-system
+    condition: '%resource.system = "email"'
+    value: [ '"NET"' ]
+    hl7Spec: [ '%{hl7TelecomPath}-2' ]
+
+  - name: il-message-encoding-characters
+    condition: 'true'
+    value: [ '"^~\\&"' ]
+    hl7Spec: [ 'MSH-2' ]
+
+  - name: il-name-of-the-sending-application
+    condition: 'true'
+    value: [ '"CDC PRIME"' ]
+    hl7Spec: [ 'MSH-4-1' ]
+
+  - name: il-OID-of-the-sending-application
+    condition: 'true'
+    value: [ '"11D2030855"' ]
+    hl7Spec: [ 'MSH-4-2' ]
+
+  - name: il-indicates-that-5.2-is-an-OID
+    condition: 'true'
+    value: [ '"CLIA"' ]
+    hl7Spec: [ 'MSH-4-3' ]
+
+  - name: il-name-of-the-receiving-application
+    condition: 'true'
+    value: [ '""' ]
+    hl7Spec: [ 'MSH-5-1' ]
+
+  - name: il-OID-of-the-receiving-application
```

Comment:
Why are we suppressing all fields from MSH-5 and MSH-6? I don't see that in their existing settings.

Reply:
Here is the setting above.        
        receivingApplicationName: null
        receivingApplicationOID: null
        receivingFacilityName: null
        receivingFacilityOID: null